ipa-core/src/query/runner/oprf_ipa.rs

```diff
@@ -69,7 +74,19 @@ where
             v.truncate(sz);
             v
         } else {
-            panic!("Encrypted match key handling is not handled for OPRF flow as yet");
+            LengthDelimitedStream::<EncryptedOprfReport<BA8, BA3, BA20, _>, _>::new(input_stream)
```

Comment:
all of these types will likely be runtime parameters. we should probably add a todo/issue to fix it later

Reply:
Agreed, OprfIpaQuery doesnt have generics for it yet. I have opened an issue https://github.com/private-attribution/ipa/issues/917